BenchmarkDotNet.Artifacts/BenchmarkRun-20250218-165908.log

```diff
@@ -0,0 +1,9 @@
+// Validating benchmarks:
```

Comment:
Are these .log files needed to belong to the repository. Isn't better to add them to the .`gitignore`?

Reply:
I agree. I added an exclude for `logs/` but I'll also add `*.log`